src/js/components/sidebar.jsx

```diff
@@ -310,6 +361,12 @@ export default class SideBar extends React.Component {
     return newTabsById;
   }
 
+  _toggleSelectClick(event) {
+    const state = this.state;
+    state.selectByClick = event.target.checked;
+    this.setState(Object.assign({}, this.state));
```

Comment:
The rest code that calls `setState` is currently incorrect as you don't need to re-assign the entire state. You can update a sub-tree and React will do the right thing. So this function becomes:

```javascript
_toggleSelectClick(event) {
    this.setState({ selectByClick: event.target.checked });
}
```